restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -660,12 +660,13 @@ public Response getDocPath(
             logger.debug(json);
             return Response.ok(json).build();
           } else {
+            int pageSize = Math.max(1, pageSizeParam);
```

Comment:
What about ` defaultValue = "100"` for the `page-size` HTTP parameter on line 587? Why is it not effective? Is it just for swagger documention? In the latter case I think we ought to fix it too :)

Reply:
Oh yes - my fault - that is just for swagger, will change